README.md

```diff
@@ -163,12 +163,14 @@ Use `firebase:deploy --only remoteconfig` to update and publish a project's Fire
 
 ### General
 
-The Firebase CLI can use one of three authentication methods listed in descending priority:
-
-- `GOOGLE_APPLICATION_CREDENTIALS` - if this environment variable points to a service account key file, the Firebase CLI will authenticate as a service account.
-- `firebase login` - you can log in to the CLI directly as yourself. The CLI will store an authorized user credential.
-  - `FIREBASE_TOKEN` - you can explicitly use this environment variable to pass in a long-lived user token from `firebase login:ci`.
-- `gcloud auth application-default login` - if your development machine has application default credentials from the Google Cloud CLI, we will use them if none of the above credentials are present.
+The Firebase CLI can use one of four authentication methods listed in descending priority:
+
+- **User Token** - provide an explicit long-lived Firebase user token generated from `firebase login:ci`. Note that these tokens are extremely sensitive long-lived credentials and are not the right option for most cases. Consider using service account authorization instead. The token can be set in one of two ways:
+  - Set the `--token` flag on any command, for example `firebase --token="<token>" projects:list`.
+  - Set the `FIREBASE_TOKEN` environment variable.
+- **Local Login** - run `firebase login` to log in to the CLI directly as yourself. The CLI will cache an authorized user credential on your machine.
+- **Service Account** - set the `GOOGLE_APPLICATION_CREDENTIALS` environment variable to point to the path of a JSON service account key file.
+- **Application Default Credentials** - if you use the `gcloud` CLI and log in with `gcloud auth application-default login`, the Firebase CLI will use them if none of the above credentials are present.
```

Comment:
Looks like this changes leaked from #2792

Reply:
@rosariopfernandes good catch!